src/ProfessionalGraveyard.tsx

```diff
@@ -63,12 +112,20 @@ export function ProfessionalGraveyard(props: GraveyardProps) {
                     alt='Fire'
                 />
             </div>
-
             <span className="uppercase text-2xl">
                 RIP you slimey bastards
             </span>
-
-            {data}
+            <button className="pt-10"onClick={handleToggleFilter}>
```

Comment:
```suggestion
            <button className="mt-10" onClick={handleToggleFilter}>
```

Margin is more appropriate and should work the same here